README.adoc

```diff
@@ -4,21 +4,57 @@ Mobile Core
 
 image:https://img.shields.io/:license-Apache2-blue.svg[License, link=http://www.apache.org/licenses/LICENSE-2.0]
 
+AeroGear on OpenShift is an project that enables set of the mobile enabled services that run natively on Open Shift platform.
+This repository is central element of the AeroGear on OpenShift ecosystem that installs Open Shift platforms and all AeroGear related templates (Ansible APB) to local machine. Developers then can use mobile-cli that is designed to work with running Open Shift instance and perform various tasks and one of the available mobile SDK's. For complete list of the repositories see bellow.
```

Comment:
This repository is central element of the AeroGear on OpenShift ecosystem that installs the OpenShift platform and the AeroGear components to your local machine. You can then use the OpenShift UI or mobile-cli to work with running Open Shift instance in order to deploy mobile enabled services and configure your mobile client to be able to use those services.